common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -353,62 +353,31 @@ def manage_video_subtitles_save(item, user, old_metadata=None, generate_translat
 
     `old_metadata` contains old values of XFields.
 
-    # 1.
-    If value of `sub` field of `new_item` is different from values of video fields of `new_item`,
-    and `new_item.sub` file is present, then code in this function creates copies of
-    `new_item.sub` file with new names. That names are equal to values of video fields of `new_item`
-    After that `sub` field of `new_item` is changed to one of values of video fields.
-    This whole action ensures that after user changes video fields, proper `sub` files, corresponding
-    to new values of video fields, will be presented in system.
-
-    # 2 convert /static/filename.srt  to filename.srt in self.transcripts.
+    # 1. convert /static/filename.srt  to filename.srt in self.transcripts.
     (it is done to allow user to enter both /static/filename.srt and filename.srt)
 
-    # 3. Generate transcripts translation only  when user clicks `save` button, not while switching tabs.
+    # 2. Generate transcripts translation only  when user clicks `save` button, not while switching tabs.
     a) delete sjson translation for those languages, which were removed from `item.transcripts`.
         Note: we are not deleting old SRT files to give user more flexibility.
     b) For all SRT files in`item.transcripts` regenerate new SJSON files.
         (To avoid confusing situation if you attempt to correct a translation by uploading
         a new version of the SRT file with same name).
     """
-
     _ = item.runtime.service(item, "i18n").ugettext
 
     # 1.
-    html5_ids = get_html5_ids(item.html5_sources)
```

Comment:
@muhammad-ammar I would say let it commented out. We will remove as a part of cleanup.

Reply:
we may look at it in future for scenarios in case if something get messed up